### ISSUE TITLE
can't enable the plugin on windows

### ISSUE THREAD

**Issue by frudas24** [OGRECave/blender2ogre#29]
Hi,
when activating the plugin in win10 it show this errors:
tested with blender 2.79 and 2.75

```
Exception while reading windows registry: [WinError 2] El sistema no puede encontrar el archivo especificado
Traceback (most recent call last):
  File "C:\Program Files\Blender Foundation\Blender\2.79\scripts\modules\addon_utils.py", line 331, in enable
    mod = __import__(module_name)
  File "C:\Program Files\Blender Foundation\Blender\2.79\scripts\addons\io_ogre\__init__.py", line 36, in <module>
    from . import properties
  File "C:\Program Files\Blender Foundation\Blender\2.79\scripts\addons\io_ogre\properties.py", line 5, in <module>
    load_user_materials()
  File "C:\Program Files\Blender Foundation\Blender\2.79\scripts\addons\io_ogre\ogre\material.py", line 405, in load_user_materials
    scripts,progs = update_parent_material_path( config.get('USER_MATERIALS') )
  File "C:\Program Files\Blender Foundation\Blender\2.79\scripts\addons\io_ogre\ogre\material.py", line 673, in update_parent_material_path
    parse_material_and_program_scripts( path, scripts, progs, missing )
  File "C:\Program Files\Blender Foundation\Blender\2.79\scripts\addons\io_ogre\ogre\material.py", line 444, in parse_material_and_program_scripts
    parse_material_and_program_scripts( url, scripts, progs, missing )
  File "C:\Program Files\Blender Foundation\Blender\2.79\scripts\addons\io_ogre\ogre\material.py", line 466, in parse_material_and_program_scripts
    p = OgreProgram( data='\n'.join(chk) )
NameError: name 'OgreProgram' is not defined
```

**Comment by paroj** [comment]
try setting USER_MATERIALS to "" (empty) in the plugin settings

**Comment by frudas24** [comment]
now i have this error:

INFO:root:target_path C:\Users\Pao Barrios\Documents\Blender\Ogre_Export_Demo\Export
INFO:root:target_file_name cube_simple_2.scene
INFO:root:target_file_name_no_ext cube_simple_2
Processing Scene: name:cube_simple_2, path: C:\Users\Pao Barrios\Documents\Blender\Ogre_Export_Demo\Export
  Processing Materials
      - Exporting root node: root
      - Generating: Cube.mesh.xml
      - Writing shared geometry
        Done at 0.02 seconds
      - Writing submeshes
        Done at 0.02 seconds
      - Created .mesh.xml at 0.03 seconds
Unknown input type.
Traceback (most recent call last):
  File "C:\Users\Pao Barrios\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\io_ogre\ui\export.py", line 101, in execute
    scene.dot_scene(target_path, target_file_name_no_ext)
  File "C:\Users\Pao Barrios\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\io_ogre\ogre\scene.py", line 159, in dot_scene
    xmlparent = doc._scene_nodes
  File "C:\Users\Pao Barrios\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\io_ogre\ogre\scene.py", line 572, in dot_scene_node_export
    xmlparent=o
  File "C:\Users\Pao Barrios\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\io_ogre\ogre\scene.py", line 572, in dot_scene_node_export
    xmlparent=o
  File "C:\Users\Pao Barrios\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\io_ogre\ogre\scene.py", line 466, in dot_scene_node_export
    mesh.dot_mesh(ob, path, overwrite=overwrite)
  File "C:\Users\Pao Barrios\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\io_ogre\ogre\mesh.py", line 620, in dot_mesh
    util.xml_convert(target_file, has_uvs=dotextures)
  File "C:\Users\Pao Barrios\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\io_ogre\util.py", line 84, in xml_convert
    assert ret == 0, "OgreXMLConverter failed"
AssertionError: OgreXMLConverter failed

location: <unknown location>:-1

location: <unknown location>:-1

**Comment by paroj** [comment]
see #21

**Comment by frudas24** [comment]
it works but with 0.7 tagged version not the master

**Comment by paroj** [comment]
original issue fixed by f5d0963e76fc9adca0274b1fde04c43a08ddd032
